includes/class-piklist-add-on.php

```diff
@@ -218,17 +218,19 @@ public static function is_active($add_on = '')
   public static function current()
   {
     $backtrace = debug_backtrace();
-    
     foreach ($backtrace as $trace)
     {
-      if (strstr($trace['file'], '/parts/'))
+	  $file = isset($trace['file']) ? $trace['file'] : '';
+	  $parts = DIRECTORY_SEPARATOR . "parts" . DIRECTORY_SEPARATOR;
+	  
+      if (strstr($file, $parts))
       {
-        $add_on = substr($trace['file'], 0, strpos($trace['file'], '/parts/'));
-        $add_on = substr($add_on, strrpos($add_on, '/') + 1);
+        $add_on = substr($file, 0, strpos($file, $parts));
+        $add_on = substr($add_on, strrpos($add_on, DIRECTORY_SEPARATOR) + 1);
 
         if (isset(piklist::$add_ons[$add_on]))
         {
-          return $add_on;
+          return str_replace(DIRECTORY_SEPARATOR, '/', $add_on);
```

Comment:
Looking at this line makes me wonder if upstream, when `piklist::$add_ons` is being built, if we should be using the DIRECTORY_SEPARATOR constant at that time. I'd prefer to work with paths as they ought to be on the server, consistently, rather than replacing separators downstream.